depends/packages/boost.mk

```diff
@@ -4,6 +4,11 @@ $(package)_download_path=https://boostorg.jfrog.io/artifactory/main/release/$($(
 $(package)_file_name=boost_$(subst .,_,$($(package)_version)).tar.bz2
 $(package)_sha256_hash=fc9f85fc030e233142908241af7a846e60630aa7388de9a5fafb1f3a26840854
 
+define $(package)_preprocess_cmds
+  rm boost/filesystem* -rf &&\
+  sed -i.old 's/\(#include <\)boost\/filesystem[^>]*>/\1filesystem>/; s/boost\(::filesystem\)/std\1/g' `find boost/process -type f`
```

Comment:
sed is much cleaner, and has not been removed.

Reply:
sed+some regex is not cleaner than a well-documented patch, from upstream.

Yes it has been removed, to the furthest extent possible. As far as I'm aware, the only usages left are where we can't use patch files, because the values being substituted are only available at runtime. Clearly this isn't an instance of having to do that.